src/components/forms/AddItemForm.tsx

```diff
@@ -63,23 +76,30 @@ export function AddItemForm({ listPath, data: unfilteredListItems }: Props) {
 			return;
 		}
 
+		if (itemQuantity < 1) {
```

Comment:
praise: nice catch for if someone does edit the quantity when adding and messes up!

Reply:
Thank you for noticing! It's always nice to be able to put in "guards" or "error boundaries" or "authentication"--whatever someone would call it!